docs/changelog.rst

```diff
@@ -26,6 +26,17 @@ KSQL 5.3.0 includes new features, including:
 * ``CREATE STREAM`` and ``CREATE TABLE`` will now allow you to create the topic if it is missing.
   To do this, specify the ``PARTITIONS`` and optionally ``REPLICAS`` in the ``WITH`` clause.
 
+* ``CREATE STREAM AS SELECT ...`` and ``CREATE TABLE AS SELECT ...`` statements will use the source
+topic partitions and replica counts for the sink topic if the config properties are not set in ``WITH`` clause.
+In case of JOIN, the left hand side topic will be used.
+This deprecates ``ksql.sink.partitions`` and ``ksql.sink.replicas`` config properties.
```

Comment:
Should we also add this to the upgrade notes?

Reply:
Yes, thanks for reminding :) Updated that one too!